packages/handleRichTextBoxKeyEventNavigation/handleRichTextBoxKeyEventNavigation.js

```diff
@@ -3,18 +3,45 @@
 const TAB = 9;
 const ESCAPE = 27;
 
-export default function(e, record) {
+const TABBABLE_ELEMENT_SELECTOR = [
```

Comment:
this is from `core/a11y.ts` right? is it worth adding a comment to keep in sync?